### DISCUSSION TITLE
I don't understand which plan should I get?

### DISCUSSION THREAD

**Discussion by Bircansezgin** [supabase/supabase#28508] [Questions]
I will make a second hand sales and buying mobile app, I will keep pictures and data, not the data of houses and cars, but I do not have a lot of money, since my project is a startup, is the 25 dollar system suitable for me? I want to switch from Firebased and use Supabse. Is there a difference in usage and is there a document relationship? like.

**Comment by GaryAustin1** [comment]
You can always start off with the free plan, many use it for awhile in early production.  The $25 plan is better not just for more egress and storage but because you get backups and your instance won't pause.

Your costs will be driven by data egress and storage(data/files) most likely.  That is in an incremental billing option.  Larger compute sizes are for increasing query loads and larger tables.  You can also scale up or down on CPU sizes.

So start with Free or $25 Pro until you need more.
